pjmedia/src/pjmedia/transport_srtp_dtls.c

```diff
@@ -1082,6 +1090,152 @@ static pj_status_t parse_setup_finger_attr(dtls_srtp *ds,
     return PJ_SUCCESS;
 }
 
+/* Parse a=candidate line */
+static pj_status_t parse_cand(dtls_srtp *ds, 
```

Comment:
I don't think it is a good idea to parse ICE SDP attr by DTLS-SRTP, also it may introduce risk of maintaining the code.
IMO it is cleaner to query the valid candidates from ICE transport, it looks like ICE transport info already provides it? If not, adding API to ICE tp (or field in ICE tp info) is perhaps the proper approach.